src/components/MenuPanel/MenuPanelAppGroup.js

```diff
@@ -0,0 +1,135 @@
+import React from 'react'
+import { spring, Motion } from 'react-motion'
+import { styled, theme, spring as springConf } from '@aragon/ui'
+import MenuPanelInstance from './MenuPanelInstance'
+import color from 'onecolor'
+
+class MenuPanelAppGroup extends React.Component {
+  handleAppClick = () => {
+    if (this.props.active) return
+    const firstInstance = this.props.instances[0]
+    this.props.onActivate(
+      this.props.appId,
+      firstInstance ? firstInstance.id : ''
+    )
+  }
+  handleInstanceClick = instanceId => {
+    this.props.onActivate(this.props.appId, instanceId)
+  }
+  render() {
+    const {
+      appId,
+      icon,
+      name,
+      instances = [],
+      active,
+      activeInstance,
+    } = this.props
+    const image = icon || <img src={`/apps-icons/${appId}.svg`} alt="" />
+    return (
+      <Motion
+        style={{
+          openProgress: spring(Number(active), springConf('fast')),
+        }}
+      >
+        {({ openProgress }) => (
+          <Main active={active}>
+            <MenuItemBar
+              style={{
+                transform: `translateX(-${(1 - openProgress) * 100}%)`,
+                opacity: openProgress,
+              }}
+            />
+
+            <ActiveBackground style={{ opacity: openProgress }} />
+
+            <a
+              role="button"
+              tabIndex="0"
+              className="item"
+              onClick={this.handleAppClick}
+            >
+              <span className="icon">{image}</span>
+              <span className="name">{name}</span>
+            </a>
+
+            <ul
+              className="instances"
+              style={{
+                display: instances.length > 0 ? 'block' : 'none',
+                height: `${(instances.length * 30 + 10) * openProgress}px`,
+                paddingBottom: `${10 * openProgress}px`,
+              }}
+            >
+              {instances.map(({ id, name }) => (
+                <li key={name}>
+                  <MenuPanelInstance
+                    id={id}
+                    name={name}
+                    active={id === activeInstance}
+                    onClick={this.handleInstanceClick}
+                  />
+                </li>
+              ))}
+            </ul>
+          </Main>
+        )}
+      </Motion>
+    )
+  }
+}
+
+const Main = styled.div`
+  position: relative;
+  width: 100%;
+  padding: 0 10px 0 30px;
+
+  .item {
+    display: flex;
+    align-items: center;
+    height: 40px;
+    cursor: pointer;
+  }
+  .name {
+    font-weight: ${({ active }) => (active ? '800' : '400')};
+  }
+  .icon {
+    display: flex;
+    width: 22px;
+    height: 22px;
+    margin-right: 15px;
+    color: ${({ active }) =>
+      active ? theme.textPrimary : theme.textSecondary};
```

Comment:
Yeah, it the hover state only works to the wrapper links (home, settings) now that icons are loaded externally for apps.

About inlining the SVGs, I would like to avoid inserting anything coming from apps in the wrapper, so that we don’t have to deal with security issues. But if an app icon follow our guidelines, its icon will be monochromatic, and it should be easy for us to generate another color from it, then we could just swap alternate between them on hover.

I will disable the hover state on the menu icons for now. 👍